python/lsst/pipe/tasks/assembleCoadd.py

```
@@ -475,17 +602,24 @@ def assembleMetadata(self, coaddExposure, tempExpRefList, weightList):
 
     def assembleSubregion(self, coaddExposure, bbox, tempExpRefList, imageScalerList, weightList,
                           bgInfoList, altMaskList, statsFlags, statsCtrl):
-        """Assemble the coadd for a sub-region
-
-        @param coaddExposure: The target image for the coadd
-        @param bbox: Sub-region to coadd
-        @param tempExpRefList: List of data reference to tempExp
-        @param imageScalerList: List of image scalers
-        @param weightList: List of weights
-        @param bgInfoList: List of background data from background matching
-        @param altMaskList: List of alternate masks to use rather than those stored with tempExp, or None
-        @param statsFlags: Statistic for coadd
-        @param statsCtrl: Statistics control object for coadd
+        """!
+        \brief Assemble the coadd for a sub-region.
+
+        For each coaddTempExp, check for (and swap in) an alternative mask if one is passed. If background
+        matching is enabled, add the background and background variance from each coaddTempExp. Remove mask
+        planes listed in config.removeMaskPlanes (default 'CROSSTALK', 'NOT_DEBLENDED'), Finally, mean-stack
```

I don't think you need to list the default mask planes to be removed.